### DISCUSSION TITLE
Which Entity to Report 'Plan Name' - Network vs Health Plan Issuer

### DISCUSSION THREAD

**Discussion by AJLgeHG** [CMSgov/price-transparency-guide#388] [General]
Hi - Plan name is required for both the In-Network (INN) file and the Table of Contents (TOC) file. We contract with different networks to provide coverage for our members. As such, these networks will provide us the INN MRF and we will create a TOC file which will have information for the INN MRFs.  The networks will not provide Plan_id_type (HIOS/EIN), plan_id (HIOS/EIN value), and plan_market_type. We have this information and will provide it in the TOC file. 

On our end the HIOS and EINs are aligned with Plan names that are not the same as those from the network. The network Plan Names are the labels they assign to their plan offerings to our members and our Plan Names are tied to the payer entity/employer groups with whom we contract to provide coverage for our members. For both the networks and us the plan names are different but correct for each of us.
 
So, the question is, which Plan Name should be used for the INN MRF? Should it be ours (the payer/employer groups) or the network's (their designated name for the plans they offer to members)?  

@shaselton-usds   please provide guidance.

Thanks

**Comment by cnicdcr** [comment]
@AJLgeHG - have you been able to get this answered?  If so, please share and the source of the answer.  Thank you
